projects/components/src/checkbox/checkbox.component.ts

```diff
@@ -12,8 +12,8 @@ import { MatCheckboxChange } from '@angular/material/checkbox';
       (change)="onCheckboxChange($event)"
       [disabled]="this.disabled"
       class="htc-checkbox"
-      (click)="$event.stopPropagation()"
-      ><htc-label class="label" *ngIf="this.label" [label]="this.label"></htc-label>
+    >
```

Comment:
This concerns me a bit - have we looked through the usages?

Reply:
right. I will do that or make it configurable. Just trying to get the table select working first